### PR TITLE
Makes toilets and sinks constructable

### PR DESCRIPTION
**Pull request by defiantGrace**
## About The Pull Request
Makes toilets and sinks constructable

image

## Why It's Good For The Game
I took a poo in a plasma toilet and that's just peachy

## Changelog
:cl:
add: makes toilets constructable
add: makes sinks constructable
/:cl:

### CODE REVIEW

File: code/game/objects/structures/watercloset.dm [hunk 1]
```diff
@@ -73,6 +74,15 @@
 /obj/structure/toilet/update_icon()
```
Comment (ninjanomnom) [inline code/game/objects/structures/watercloset.dm]: Switch this to update_icon_state while you're here
```suggestion
/obj/structure/toilet/update_icon_state()
```

**Review by ninjanomnom — CHANGES_REQUESTED** [review]

**Comment by MCterra10** [comment]
Good first pr 👍

**Comment by ArcaneMusic** [comment]
Not even just constructable, but constructable with datum mats too? Nice!

**Comment by MCterra10** [comment]
Please make it so that we can buckle people to toilets. I want to buckle people to uranium toilets and irradiate them

**Comment by fludd12** [comment]
Next is to make them and showers valid plumbing connections

**Comment by TheChosenEvilOne** [comment]
Should shift the sinks a bit i'd say, they don't look that great when separated from the wall.

**Comment by carlarctg** [comment]
plasma shower please

**Comment by dr-bee** [comment]
Time to make that solid gold toilet ive always wanted.

**Pull request merged**